lib/config-generator.js

```diff
@@ -148,11 +148,25 @@ class ConfigGenerator {
             entry[sharedEntryTmpName] = tmpFilename;
         }
 
-        if (this.webpackConfig.copyFilesConfigs.length > 0) {
+        const copyFilesConfigs = this.webpackConfig.copyFilesConfigs.filter(entry => {
+            const copyFrom = path.resolve(
+                this.webpackConfig.getContext(),
+                entry.from
+            );
+
+            if (!fs.existsSync(copyFrom) || !fs.lstatSync(copyFrom).isDirectory()) {
```

Comment:
I guess we could break these into 2 separate pieces: 1 error is that the path doesn't exist. Another error is that they've pointed directly to a single file, not a directory, right?

Reply:
👍